guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -32,6 +39,9 @@ endif::[]
 endif::[]
 
 ifdef::satellite[]
+For detailed information on partitioning and size, refer to the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/system_design_guide/partitioning-reference_system-design-guide[{RHEL} 8 partitioning guide].
```

Comment:
Is this a good-to-go @ehelms ? The link you are asking for.

Reply:
The link reads good, I don't think we have to gate this on `ifdef::satellite[]` though, as this is useful to any deployment on EL8. So I would drop that ifdef wrapper around this.

Reply:
Addressed :+1: